### ISSUE TITLE
[Feature Request] Collapsible Buff/Debuffs similar to what the default UI got in the rework

### ISSUE THREAD

**Issue by ThePrimoris** [tukui-org/ElvUI#807]
**Describe the solution you'd like**
Ability to collapse the buff/debuff and choose the flyout direction so they float left/right.

**Comment by Doosty** [comment]
Any progress on this? These days we don't need to have players and targets buffs on screen 24/7 it is just visual clutter salad...

**Comment by Azilroka** [comment]
I assume you mean the ones by the minimap? Those are never your target debuffs.

**Comment by Doosty** [comment]
Yes this thread is about the buffs/debuffs next to the minimap which the blizzard ui can collapse like this -> 
![image](https://github.com/user-attachments/assets/d0dcaacf-658d-45a1-b75c-a37a2120f87a)
However i would argue that if this feature is implemented in elvui it should also be extended to the **target** buff/debuff frames as they are cluttering the ui in the same way imo.

**Comment by Repooc** [comment]
Considering u only see target auras on the target unitframe, and you can filter those down to nothing if you want, a collapsing button wont happen for other frames including the target unitframe.
The only reasons the one by the minimap is even being considered is bc the default ui has it and you can't filter the auras by the minimap.
